backend/manager/views.py

```diff
@@ -58,6 +59,12 @@ class CrawlerList(generics.ListCreateAPIView):
     permission_classes = (IsAdminUser, )
 
     def perform_create(self, serializer):
+        crawlers = Crawler.objects.all()
```

Comment:
You should not raise `ValidationError` at `perform_create` because this invokes DEBUG mode page.
In actual product environment, the DEBUG page will not be shown.
Rather, there will be only 500 Internal Server Error message.
So, remove this code and use same mechanism to serializer.
Please refer to `StudentSerializer`.
There is `validate` function, which is helpful for raising validation error correctly.